### ISSUE TITLE
Exchange MVC Projects Sign Transactions

### ISSUE THREAD

**Issue by fcoskun** [LedgerHQ/ledger-dotnet-api#7]
Hi,

Is there any way to sign and create transaction on mvc server app when i use my ledger on my browser. i see these description but i dont know how we build up. 

Use a custom transport protocol to talk to your ledger (Example: having a HTTP proxy to talk to a remote ledger connected to your server)

Thanks,

**Comment by NicolasDorier** [comment]
hey yes, though I have not made a lib for this, but the basic idea is:

On client side I use this:

https://github.com/btcpayserver/ledgerwebsocket

This is a library which plug the U2F interface of the ledger to a websocket exposed [here](https://github.com/btcpayserver/btcpayserver/blob/08badbde56cc9d640b4fd1654b0113894c95bd41/BTCPayServer/Controllers/StoresController.BTCLike.cs#L171) by the back end. (I call this javascript component `the bridge`)

Then on server side, I change the `ITransport` implementation of the `LedgerClient` with one which send the APDU to this websocket. [Implementation here](https://github.com/btcpayserver/btcpayserver/blob/806474c8c6734f7f2ba94d4cc5b49f2f3dbe85fb/BTCPayServer/Services/HardwareWalletService.cs#L24)

Then, javascript side I can send commands to the server to the same websocket as you can see [Here](https://github.com/btcpayserver/btcpayserver/blob/08badbde56cc9d640b4fd1654b0113894c95bd41/BTCPayServer/wwwroot/js/StoreWallet.js).

In a nutshell: I use the `LedgerClient` server side, changed the transport so the data is sent to websocket, and a component (the bridge) is forwarding this to the U2F interface of ledger.
